code/__HELPERS/roundend.dm

```diff
@@ -170,10 +170,16 @@
 		cb.InvokeAsync()
 	LAZYCLEARLIST(round_end_events)
 
+	var/speed_round = FALSE
+	if(world.time - SSticker.round_start_time <= 90 SECONDS)
+		speed_round = TRUE
+
 	for(var/client/C in GLOB.clients)
 		if(!C.credits)
 			C.RollCredits()
 		C.playtitlemusic(40)
+		if(speed_round)
+			C.give_award(/datum/award/achievement/misc/speed_round, user)
```

Comment:
Is this really a good idea? This is going to encourage revheads to instantly suicide to get achievements

Reply:
I can raise the number a bit, I doubt people will be that autistic but nothing surprised me by now either.

Reply:
I would raise it to like 5 minutes at least

Reply:
If revheads instantly suicide that's an administrative issue, pretty egregious thing to do that would probably result in being banned from the role

Reply:
make this only give out the medal to greentexting antags (NOT to ordinary crewmembers or to redtexting antags)